### ISSUE TITLE
Filetype selection for new file

### ISSUE THREAD

**Issue by santimonti** [geany/geany#2495]
Hey, first of all, sorry for my bad English.
Every time I create a new file I have to select the file type this way: document->set filetype->.....

And it's annoying. Its there a shortcut or a custom command I can use to select programming language to pascal fast?

**Comment by elextr** [comment]
No, but if you save the file with a recognised pascal extension it should select the filetype then.

**Comment by santimonti** [comment]
> No, but if you save the file with a recognised pascal extension it should select the filetype then.

So its no way I can set the filetype easier right?

**Comment by elextr** [comment]
Well, you could create a template for a pascal file, see [the manual](https://www.geany.org/manual/current/index.html#file-templates), and create it with `New (from template)`
